src/main/java/com/ericsson/ei/queryservice/ProcessQueryParams.java

```diff
@@ -123,14 +126,15 @@ private JSONArray filterResult(String filter, JSONArray resultAggregatedObjectAr
                 resultArray.put(tempJson);
             }
         } catch (JSONException e) {
-            LOGGER.error("Failed to filter an object\n: " + e.getMessage());
+            LOGGER.error("Failed to filter an object: \n" + e.getMessage());
```

Comment:
Remove whitespace between ':' and '\n'

Reply:
I'll remove the newline

Reply:
Depends if its a long string, the its is nicer with starting on a new fresh line. Thats my opinion.

Reply:
True in this case it is a long JSON strcutre being returned so maybe it's better to have it on a separate line